### ISSUE TITLE
Getting blank response when starting the installation bash script.

### ISSUE THREAD

**Issue by shubhankartrivedi** [pterodactyl-installer/pterodactyl-installer#428]
Getting blank response when starting the installation bash script.

Steps to reproduce the behaviour:

1. Install RaspberryPi OS on RPI 3 B+.
2. Do the basic upgrade scripts.
3. Get to the root.
4. Start the installation script.

**Screenshots**
<img width="496" alt="Screenshot 2023-08-20 at 4 18 33 PM" src="https://github.com/pterodactyl-installer/pterodactyl-installer/assets/78972053/5924f401-38dc-4de0-a23b-7a5b5d4ca981">

- OS: Debian (RPI OS)

- Installation: Both

**Additional context**
So basically it is stuck and gives no response.
I have tried reinstalling everything from bash to the OS and nothing works.

**Comment by Linux123123** [comment]
Try starting using this command
```
wget https://pterodactyl-installer.se/i.sh
bash i.sh
```

**Comment by shubhankartrivedi** [comment]
> Try starting using this command
> 
> ```
> wget https://pterodactyl-installer.se/i.sh
> bash i.sh
> ```

Just did! It works now!

**Comment by shubhankartrivedi** [comment]
It was working but it got stuck again....?

<img width="567" alt="Screenshot 2023-08-20 at 4 35 20 PM" src="https://github.com/pterodactyl-installer/pterodactyl-installer/assets/78972053/1a2f64fd-cc0f-4860-85d5-c999a174996b">

**Comment by shubhankartrivedi** [comment]
Everything works as you can see but the problem is on script execution only.

<img width="123" alt="Screenshot 2023-08-20 at 4 41 53 PM" src="https://github.com/pterodactyl-installer/pterodactyl-installer/assets/78972053/9a968e73-2e96-4fc9-9655-a0cb07190397">

**Comment by shubhankartrivedi** [comment]
Here is more version info:

<img width="353" alt="Screenshot 2023-08-20 at 5 11 43 PM" src="https://github.com/pterodactyl-installer/pterodactyl-installer/assets/78972053/7d6ef92b-8c05-43a6-b8f4-6cbf1ea37a19">

**Comment by vilhelmprytz** [comment]
This is not a problem with the script. Please check that you can reach GitHub from your server.

**Comment by shubhankartrivedi** [comment]
> This is not a problem with the script. Please check that you can reach GitHub from your server.

I solved the issue easily by doing some DNS edits. It was a network issue.
